src/tests/multihost/sssd/testlib/common/expect.py

```diff
@@ -40,6 +41,22 @@ def login(self, login_timeout=10, auto_prompt_reset=True,
         except pexpect.pxssh.ExceptionPxssh:
             raise SSHLoginException("%s Failed to login" % self.username)
 
+    def fast_login(self):
+        ssh = ctypes.CDLL("./library.so")
```

Comment:
What is `library.so`?
Is it compiled from `remote_ssh.go` somehow? (how?)

Reply:
I talking to idm-ops guys so that  library.so file can be generated before we start the test from  remote_ssh.go file

Reply:
ok good.  That was my first thought too.  Can we have the library built beforehand and then just referenced from a specific location where it's installed?  /tmp/library.so seems a little too generic.

Reply:
yes test runs are already passing and it has been moved to /tmp//library.so

Reply:
Can it be named something more descriptive?

Reply:
i can rename it as ssh_library.so

Reply:
Thanks.  Looks good.